book/07-git-tools/sections/signing.asc

```diff
@@ -199,5 +199,6 @@ Merge made by the 'recursive' strategy.
 ==== Everyone Must Sign
 
 Signing tags and commits is great, but if you decide to use this in your normal workflow, you'll have to make sure that everyone on your team understands how to do so.
+This can be achieved by asking everyone working with the repository to run `git config --local commit.gpgsign true` to automatically have all of their commits in the repository.
```

Comment:
```suggestion
This can be achieved by asking everyone working with the repository to run `git config --local commit.gpgsign true` to automatically have all of their commits in the repository signed by default.
```

LGTM except one possible word missing at the end maybe? I'm not native speaker though.

Reply:
Indeed - updated and rebased on latest main.